src/components/TransactionsTracker/useTransactionsTracker.ts

```diff
@@ -0,0 +1,43 @@
+import { useEffect, useRef } from 'react';
+import { getTransactionsByHashes as defaultGetTxByHash } from 'apiCalls/transactions';
+import { GetTransactionsByHashesType } from 'contexts/types';
+import { useCheckTransactionStatus } from 'hooks';
+import { useGetPendingTransactions } from 'services/transactions';
+import { getIsTransactionPending } from 'utils';
+
+export interface TransactionsTrackerType {
+  getTransactionsByHash?: GetTransactionsByHashesType;
+}
+
+export function useTransactionsTracker(props?: TransactionsTrackerType) {
+  const { pendingTransactionsArray } = useGetPendingTransactions();
+
+  const getTransactionsByHash =
+    props?.getTransactionsByHash ?? defaultGetTxByHash;
```

Comment:
There is already a fallback on the default, either leave it only here it only in the component above

Reply:
✅